Source/Core/InputCommon/ControllerInterface/ControllerInterface.cpp

```diff
@@ -207,13 +234,48 @@ void ControllerInterface::Shutdown()
 #ifdef CIFACE_USE_DUALSHOCKUDPCLIENT
   ciface::DualShockUDPClient::DeInit();
 #endif
+
+  // Make sure no devices had been added within Shutdown() in the time
+  // between checking they checked atomic m_is_init bool and we changed it.
+  // We couldn't have locked m_devices_mutex nor m_devices_pupulation_mutex for the whole Shutdown()
+  // as they could cause deadlocks. Note that this is still not 100% safe as some backends are
+  // shutdown in other places, possibly adding devices after we have shut down, but the chances of
```

Comment:
Grammatically correct but there are 3 or 4 other similar comments that already used shutdown as verb in the file, I'd keep consistency.

Reply:
I brought it up because the very next sentence fragment (correctly) uses "shut down" rather than "shutdown", which makes the whole thing even more inconsistent. Might as well fix the grammar so that the new comment is *at least* consistent with itself.